package.json

```diff
@@ -72,7 +72,8 @@
     "intl-messageformat": "^2.1.0",
     "intl-relativeformat": "^2.1.0",
     "invariant": "^2.1.1",
-    "react-display-name": "^0.2.4"
+    "react-display-name": "^0.2.4",
+    "react-is": "^16.8.6"
```

Comment:
Do you know whether or not `react-is@16.8.6` can be used with older `react` versions (`16.3.1` and upwards)?

Reply:
Good question, perhaps it is better to keep the dependency in line with the peer dependency on react? So I'll make it `^16.3.1`.